build/charts/flow-aggregator/templates/deployment.yaml

```diff
@@ -91,3 +93,9 @@ spec:
         hostPath:
           path: /var/log/antrea/flow-aggregator
           type: DirectoryOrCreate
+      # Make it optional as we only read it when customCACert=true.
+      - name: clickhouse-ca
+        secret:
+          secretName: clickhouse-ca
+          defaultMode: 0400
+          optional: true
```

Comment:
You could use
```
{{- if and .Values.clickHouse.tls.enable .Values.clickHouse.tls.customCACert }}
{{- end }}
```
to wrap this instead of using the optional feature.
And same for the `clickhouse-ca` VolumeMounts above.

Reply:
Should we keep `optional` here? I think if user doesn't deploy clickhouse-ca secret, the flow-aggregator can't neither start or log the error?

Reply:
I got your point. Please feel free to ignore this comment.

Reply:
Yes.

Reply:
I think when tls is disabled, we could still wrap it to make it clear?